x-pack/plugins/security_solution/public/flyout/document_details/right/components/header_actions.tsx

```diff
@@ -82,8 +76,8 @@ export const HeaderActions: VFC = memo(() => {
                     { defaultMessage: 'Share alert' }
                   )}
                   data-test-subj={SHARE_BUTTON_TEST_ID}
-                  onClick={() => copyFunction(copy, alertDetailsLink, modifier)}
-                  onKeyDown={() => copyFunction(copy, alertDetailsLink, modifier)}
+                  onClick={() => copyFunction(copy, alertDetailsLink)}
```

Comment:
nit: if url modification is no longer necessary, we probably don't need the [helper function](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/shared/utils/copy_to_clipboard.tsx#L18) and just use eui copy?